src/commands/lambda/functions/commons.ts

```diff
@@ -76,6 +112,36 @@ export const getLambdaFunctionConfigs = (
   return Promise.all(promises)
 }
 
+/**
+ * Returns the correct ARN of a **Specific Runtime Layer** given its configuration, region,
+ * and settings (optional).
+ *
+ * @param config a Lambda FunctionConfiguration.
+ * @param region a region where the layer is hosted.
+ * @param settings instrumentation settings, mainly used to change the AWS account that contains the Layer.
+ * @returns the ARN of a **Specific Runtime Layer** with the correct region, account, architecture, and name.
+ */
+export const getLayerArn = (
```

Comment:
it seems that this function looks a lot like `getExtensionArn`? Maybe we should refactor?

Reply:
I refactored the function to be able to get the Extension, let me know what you think about the followed approach.